MapboxVision/VisionManager/VisionReplayManager.swift

```diff
@@ -24,7 +25,7 @@ public final class VisionReplayManager: BaseVisionManager {
         
         - Parameter recordPath: Path to a folder with recorded session. You typically record such sessions using `startRecording` / `stopRecording` on `VisionManager`.
 
-        - Returns: Instance of `VisionRecordManager` configured to use data from specified session.
+        - Returns: Instance of `VisionReplayManager` configured to use data from specified session.
```

Comment:
Should we add info about how to gracefully destroy instance?
Like `You must call destroy() method...` or smith like this.`

Reply:
Please propose in https://github.com/mapbox/mapbox-vision/pull/1042/. This is just a mirror.
Concerning particular thing we've added lifecycle to the top of the class.